### DISCUSSION TITLE
Updates

### DISCUSSION THREAD

**Discussion by T1xx1** [abbrcode/abbreviations-in-code#93] [Announcements]
Instead of writing our updates in the chat discussion let's write them here.

**Comment by T1xx1** [comment]
Completed [task](https://github.com/orgs/abbrcode/projects/1/views/2?pane=issue&itemId=22451009)

**Comment by oood** [comment]
Sorry guys, I've been super busy this week with a new project to release.

I'll try to spend as much time here as I can, but contributions can be slow, sorry.

**Reply by T1xx1** [reply]
Don't worry I can guide the project while your busy 😉.
I haven't done much progress yet so you didn't loose anything important.

**Reply by oood** [reply]
Thank you very much, I really don't have a lot of time this week, I'm seriously behind schedule on another project, and I'm chasing the release date.

**Comment by T1xx1** [comment]
Today I'm trying to create the org .github repo profile

**Comment by T1xx1** [comment]
Merged [#54](https://github.com/abbrcode/abbreviations-in-code/pull/54)
Fixed projects descs.

**Comment by T1xx1** [comment]
![Screenshot (2)](https://user-images.githubusercontent.com/69322987/226171641-d8929e08-922d-4c1a-99f4-43618affed26.png)

**Reply by oood** [reply]
love this!

**Comment by T1xx1** [comment]
Happy 100th [commit](https://github.com/abbrcode/abbreviations-in-code/commit/5b9d326b67ccac8fef0212952caedfba924e11ec) from abbreviations in code 🎉.

**Comment by oood** [comment]
> Instead of writing our updates in the chat discussion let's write them here.

That's a good idea, but sorry I've been having a hard time these few weeks, I'll see what to do.

**Comment by T1xx1** [comment]
Added [GitHub tabs](https://github.com/abbrcode/.github/commit/f52031a72c25a8697b98146c3ba21c245c3e662a) in [.github](https://github.com/abbrcode?view_as=public).

Done [task](https://github.com/orgs/abbrcode/projects/7/views/2?pane=issue&itemId=22319867) and [task](https://github.com/orgs/abbrcode/projects/7?pane=issue&itemId=22515909)

**Comment by T1xx1** [comment]
- Closed [discussion](https://github.com/orgs/abbrcode/discussions/5)
- Abbr graph [commit](https://github.com/abbrcode/.github/commit/516fbdc29b575c8c793a84c874551a010fc864d3)
- New db [repo](https://github.com/abbrcode/db)
- New db [project](https://github.com/orgs/abbrcode/projects/5)
- Fixed repo's link to diff projects

**Comment by T1xx1** [comment]
[Db](https://github.com/abbrcode/db/blob/main/main.abbr.yml) abbrs
